sql/payday.sql

```diff
@@ -76,10 +76,24 @@ CREATE INDEX ON payday_payment_instructions (participant);
 CREATE INDEX ON payday_payment_instructions (team);
 ALTER TABLE payday_payment_instructions ADD COLUMN is_funded boolean;
 
+UPDATE payday_payment_instructions ppi
+   SET giving_due = s.giving_due
+  FROM (SELECT participant, team, SUM(giving_due) AS giving_due
+       FROM payment_instructions 
+       GROUP BY participant, team) s
+ WHERE ppi.participant = s.participant
+   AND ppi.team = s.team;
+
+DROP TABLE IF EXISTS participants_payments_uncharged;
+CREATE TABLE participants_payments_uncharged AS
+    SELECT participant 
+      FROM payday_payment_instructions
+     WHERE 1 = 2;
```

Comment:
https://github.com/gratipay/gratipay.com/pull/3675#issuecomment-131705592:

> The '1 = 2' thing is an old trick I've used quite a lot - when I wished to copy the structure (full or part) of some table, without the data. Might be overkill here for just one column, but for one thing, all other table creations in this script are in the form of selects and also, this usage means it will not break if the participant column is modified in payday_payment_instructions/payment_instructions.